### ISSUE TITLE
TextRow is not able to render Korean string correctly

### ISSUE THREAD

**Issue by karansota** [EurekaCommunity/SplitRow#38]
`SplitRow<TextRow, LocationRow> {
                $0.rowLeftPercentage = 0.65
                $0.rowLeft = TextRow("locationText") {
                    $0.placeholder = placeholder
                    if let name = value?.name {
                        $0.value = name
                    }
                    if isDisabled {
                        $0.disabled = true
                    }
                }
                $0.rowRight = LocationRow("locationLookup") {
                    $0.shouldDisplayValue = false
                    $0.title = title
                    if let value = value {
                        $0.value = value
                    } else {
                        $0.value = Place(address: nil, coordinates: nil, name: nil)
                    }
                    if isDisabled {
                        $0.disabled = true
                    }
                }
}`

Actual String when user types into the TextRow: 시ㅇ
Expected String: 설 (above characters should build this string)

<img width="370" alt="Screen Shot 2019-09-17 at 10 29 41 PM" src="https://user-images.githubusercontent.com/2958150/65114244-e9126a80-d99a-11e9-8bb1-b182050a950a.png">

**Comment by karansota** [comment]
`TextRow` when used by itself has no issues.

**Comment by marbetschar** [comment]
@karansota wow, nice catch! I wasn't even aware of the fact, that input character conversion exists at all!  :) - altough it makes perfectly sense if you think about it.

My buest guess as of now is that there is some sort of conversion-triggered-re-rendering of the underlying [UITextInput](https://developer.apple.com/documentation/uikit/uitextfield) which gets lost somewhere on its way. Will need some time figure out what exactly causes this unexpected behaviour.

Any chance you can confirm the same (wrong) behaviour for the TextAreaRow?

**Comment by marbetschar** [comment]
@mats-claassen do you have any experience with this? I don't know how `UITextField` works under the hood to render Asian text where the user input is not the actual rendered text.

Maybe you have some tips on where to start and/or why this basically works in `TextRow`, but only if it is not embedded in a `SplitRow`?

**Comment by mats-claassen** [comment]
> @mats-claassen do you have any experience with this? I don't know how UITextField works under the hood to render Asian text where the user input is not the actual rendered text.

No, I haven't. My suggestion would be to see what SplitRows does that might change how a TextRow works inside. Maybe it has to do with #39 but that is only a wild guess

**Comment by LukeGT86** [comment]
The same problem occurred when I typed in Japanese.
The suggestion appears above the keyboard when using TextRow by itself, but not in the TextRow in SplitRow.
(I think this problem is as inconvenience as I can only enter uppercase letters when I want to enter the alphabet, to me.)
Is there a solution?
![IMG_5913 (1)](https://user-images.githubusercontent.com/64204135/81791085-38686500-9541-11ea-9388-f644492820c1.jpg)     ![IMG_5914 (1)](https://user-images.githubusercontent.com/64204135/81791088-3900fb80-9541-11ea-8972-2471f6506169.jpg)
